internal/cmd/commands/server/server.go

```diff
@@ -342,6 +342,8 @@ func (c *Command) Run(args []string) int {
 			api.MeSubscriptionsHandler(cfg, c.Log, goog, db)},
 		{"/api/v1/people", api.PeopleDataHandler(cfg, c.Log, goog)},
 		{"/api/v1/products", api.ProductsHandler(cfg, algoSearch, c.Log)},
+		{"/api/v1/products", apiv2.ProductsHandler(srv)},
+		{"/api/v1/projects", apiv2.ProjectsHandler(srv)},
```

Comment:
Not sure why we would add `v1` endpoints for projects given it is a brand new feature? Is this to support some type of backwards incompatibility?

Reply:
The API version is (more or less, currently, but there's a draft PR #399 that might make this explicit) global for all frontend `fetch` calls, so I wanted to make sure that this is supported in v1 of the API as well (while it exists). The same code is used for both API versions.

Reply:
Makes sense!